lib/firespring_dev_commands/aws/route53.rb

```diff
@@ -0,0 +1,72 @@
+require 'aws-sdk-route53'
```

Comment:
The requires shouldn't be necessary - we made a change where we automatically require all of the gems included in the Gemfile in (Bundler.require call in ~/dev_commands/lib/firespring_dev_commands.rb)

Reply:
Not sure if I am missing another reference for this to function as you describe, but it won't init without the require. 

`uninitialized constant Aws::Route53`